ReqIFSharp/AttributeDefinition/AttributeDefinitionEnumeration.cs

```diff
@@ -115,6 +115,11 @@ public override void ReadXml(XmlReader reader)
         {
             base.ReadXml(reader);
 
+            if (reader.GetAttribute("MULTI-VALUED") == "true")
+            {
+                IsMultiValued = true;
```

Comment:
Please use `this.` qualifier for all public and private property, field and method usages for consistency.

Reply:
Done.